scripts/install-snapshot.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -e
+
+# This script can be used to install/delete snapshotcontroller and snapshot beta CRD
+
+SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"master"}
+
+SCRIPT_DIR="$(dirname "${0}")"
+
+TEMP_DIR="$(mktemp -d)"
+SNAPSHOTTER_URL="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOT_VERSION}"
+
+# controller
+SNAPSHOT_RBAC="${SNAPSHOTTER_URL}/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml"
+SNAPSHOT_CONTROLLER="${SNAPSHOTTER_URL}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml"
+
+# snapshot CRD
+SNAPSHOTCLASS="${SNAPSHOTTER_URL}/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml"
+VOLUME_SNAPSHOT_CONTENT="${SNAPSHOTTER_URL}/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml"
+VOLUME_SNAPSHOT="${SNAPSHOTTER_URL}/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml"
+
+function install_snapshot_controller() {
+    local namespace=$1
+    if [ -z "${namespace}" ]; then
+        namespace="default"
+    fi
+
+    create_or_delete_resource "create" $namespace
```

Comment:
still using `$namespace` and not `${namespace}`, not a major issue, but would be good to correct these cases if there is an update to this PR needed.

Reply:
sorry i missed this one, will fix it now only